### ISSUE TITLE
[Bug]: theme selection should be a dropdown/select rather than a toggle

### ISSUE THREAD

**Issue by kgryte** [stdlib-js/www#42]
### Description

The current theme selection UI does not accommodate future theme extensions, such as high contrast, in order to accommodate web accessibility needs.

Currently, the choice is binary, but we're likely to want at least three choices.

<img width="357" alt="Screen Shot 2021-10-02 at 1 17 26 PM" src="https://user-images.githubusercontent.com/2643044/135730724-a37e9bf5-c0bc-4fdb-8d22-874050c7b37a.png">

### Related Issues

None.

### Questions

No.

### Reproduction

```shell
-   open settings menu.
```

### Expected Results

```shell
-   UI element which accommodates more than 2 options.
```

### Actual Results

```shell
-    UI element which only accommodates 2 options.
```

### Environments

Firefox, Chrome, Safari, Microsoft Edge, Internet Explorer, Other Browser

### Checklist

- [X] Read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.

**Comment by Planeshifter** [comment]
While there could be other reasons to not use the MUI `<ToggleButton />`, the component does easily: accommodate more than two options. See [their documentation](https://mui.com/components/toggle-button/#main-content) for various examples.

**Comment by kgryte** [comment]
IMO, a toggle button group is still the wrong abstraction here. Say we have 10 different themes; presenting what effectively amounts to a radio button UI pattern would be unorthodox, at best, and generally unexpected.

We're unlikely to have minimal button icons representing the themes, so cannot readily follow the examples in the docs for text justification. The most compact and expected UI pattern would be a dropdown.

**Comment by kgryte** [comment]
This has now been updated to a dropdown.

![Screen Shot 2022-05-13 at 5 36 57 PM](https://user-images.githubusercontent.com/2643044/168404325-76cb36ef-b48b-4bc8-b0a3-274f0a9e9423.png)

![Screen Shot 2022-05-13 at 5 36 49 PM](https://user-images.githubusercontent.com/2643044/168404322-ef8b3f0b-a6ba-4c80-a35b-f77a361f2df4.png)
